airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -22,28 +22,28 @@
 - name: Amazon Seller Partner
   sourceDefinitionId: e55879a8-0ef8-4557-abcf-ab34c53ec460
   dockerRepository: airbyte/source-amazon-seller-partner
```

Comment:
Please leave the version change only for the connectors with which you worked in this PR, roll back the rest to the state of the master branch.